packages/cells/src/model.ts

```diff
@@ -185,7 +185,11 @@ export function isRawCellModel(model: ICellModel): model is IRawCellModel {
  */
 export class CellModel extends CodeEditor.Model implements ICellModel {
   constructor(options: CellModel.IOptions<ISharedCell> = {}) {
-    super(options);
+    super({
+      sharedModel: createStandaloneCell({ cell_type: 'raw', id: options.id }),
+      ...options
+    });
+    this._standaloneModel = typeof options.sharedModel === 'undefined';
```

Comment:
I'm not sure about this. What happens if we create a standalone Code cell?
In this case, `options.sharedModel` won't be `undefined` and `this_standalone` will be false, so when disposing the cell, we will not dispose the sharedModel, right?

I tried the console, and the cells there are not standalone.

Reply:
> That part was pushed in a separate PR addressing existing notebook memory leaks